examples/01_typescript/src/App.tsx

```diff
@@ -2,7 +2,7 @@ import React, { Suspense } from 'react'
 import { cacheExchange, createClient, fetchExchange } from '@urql/core'
 import { useAtom } from 'jotai/react'
 import { atom } from 'jotai/vanilla'
-import { atomsWithQuery } from 'jotai-urql'
+import { atomWithQuery } from '../../../src/index'
```

Comment:
The reason we prefer `jotai-urql` is to allow running the example directly in codesandbox.

Reply:
Question then. Is it possible to upload examples from the codebase up to codesandbox and keep them in sync?

Reply:
https://githubbox.com/jotaijs/jotai-urql/tree/main/examples/01_typescript
which redirects to https://codesandbox.io/s/github/jotaijs/jotai-urql/tree/main/examples/01_typescript

Reply:
Pretty cool. Didn't know that was possible.